assets/js/steps/check-in-items.js

```diff
@@ -76,6 +78,55 @@ function CheckInItems({ actionHandler }) {
         items = adaptListOfBooksToBanner(context.machineState.get.items);
     }
 
+    /**
+     * Handles numpadpresses.
+     *
+     * @param key
+     *    The pressed button.
+     */
+    function onNumPadPress(key) {
+        let typedBarcode = `${scannedBarcode}`;
+        setActiverBanner(false);
+        switch (key) {
+            case deleteButtonLabel:
+                typedBarcode = typedBarcode.slice(0, -1);
```

Comment:
setScannedBarcode(typedBarcode.slice(0, -1));

Because otherwise it will be double set with okButtonLabel